pulpcore/app/viewsets/user.py

```diff
@@ -164,10 +173,33 @@ class GroupObjectPermissionViewSet(NamedModelViewSet):
     serializer_class = PermissionSerializer
     queryset = Permission.objects.all()
 
+    def get_object_pk(self, request):
+        """Get object pk."""
+
+        if "obj" not in request.data:
+            raise ValidationError(_("Please provide 'obj' value"))
+        try:
+            obj_pk = request.data["obj"].strip("/").split("/")[-1]
+            uuid.UUID(obj_pk)
+        except (AttributeError, ValueError):
+            raise ValidationError(_(f"Invalid value for 'obj': {request.data['obj']}"))
```

Comment:
Stackoverflow says nope:
https://stackoverflow.com/questions/49797658/how-to-use-gettext-with-python-3-6-f-strings

```suggestion
            raise ValidationError(_("Invalid value for 'obj': " + str(request.data['obj'])))
```
Or the good old "format" of course (Usually this helps people translating if they see the whole string template).
```suggestion
            raise ValidationError(_("Invalid value for 'obj': {obj}").format(request.data['obj']))
```